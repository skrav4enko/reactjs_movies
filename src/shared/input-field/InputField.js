import React from 'react';
import './InputField.scss';

function InputField(props) {
  const { placeholder, value, onChange } = props;

  return <input className="input-field" placeholder={placeholder} value={value} onChange={onChange} />;
}

export default InputField;
