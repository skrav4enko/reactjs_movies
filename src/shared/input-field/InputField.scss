.input-field {
  background-image: linear-gradient(
    to top,
    rgba(0, 0, 0, 0.5) 0%,
    rgba(0, 0, 0, 0.8) 10%,
    rgba(0, 0, 0, 0.9) 50%,
    rgba(0, 0, 0, 0.8) 90%,
    rgba(0, 0, 0, 0.5) 100%
  );
  background-color: transparent;
  height: 60px;
  font-size: 20px;
  color: #ffffff;
  padding: 0 20px;
  border-radius: 4px;
  border: none;
}
