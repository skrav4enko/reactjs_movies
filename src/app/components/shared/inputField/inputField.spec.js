/* eslint-disable */
import React from 'react';
import { shallow } from 'enzyme';
import InputField from '.';

describe('InputField', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<InputField />);
    expect(wrapper).toMatchSnapshot();
  });
});

