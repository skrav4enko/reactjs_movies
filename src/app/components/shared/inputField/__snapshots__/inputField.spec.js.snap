// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputField should match snapshot 1`] = `
<input
  className="input-field"
  id="string"
  name="string"
  onChange="func"
  placeholder="string"
  type="string"
  value="string"
/>
`;
